nmigen_boards/hadbadge.py

```diff
@@ -0,0 +1,168 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ecp5 import *
+from .resources import *
+
+
+__all__ = ["HADBadgePlatform"]
+
+
+class HADBadgePlatform(LatticeECP5Platform):
```

Comment:
I see two fairly hypothetical issues here:
  1. Is this *the* hackaday badge? Aren't there one like, every year?
  2. How common is this board in the wild? I don't really plan on having any kind of popularity cutoff for now (anything that isn't an one-off and has some public presence is OK), but I'm curious how useful this pinout really is.

Reply:
1.  This is technically the Supercon19 badge; but all of their designs (and LiteX) seem to refer to it as just `hadbadge` -- probably because this is and likely will be the only FPGA one. I'd be fine with calling it `HADBadge19` or `Supercon19Badge` or anything similar; I just tend to go with whatever everyone else does.

2. There are around 500 of these currently floating around the community; and I think there's a good overlap between the group of people who attended or otherwise have one and the community of people who are likely to use nMigen. :)

Reply:
> I'd be fine with calling it HADBadge19 or Supercon19Badge or anything similar; I just tend to go with whatever everyone else does.

LMK what you prefer and I'll do pretty much whatever. :)

Reply:
`Supercon19Badge` seems like it's about as descriptive and unambiguous as it gets.

Reply:
Cool; that works.